plugins/jenkins-backend/src/types.ts

```diff
@@ -63,7 +63,7 @@ export interface BackstageBuild extends CommonBuild {
 
 export interface CommonProject {
   // standard Jenkins
-  lastBuild: CommonBuild;
+  lastBuild: CommonBuild | null;
```

Comment:
I'm also thinking that we should avoid the use of null in our representation of these, so these and the other ones would instead be `lastBuild?: CommonBuild`. If you feel it's needed we could create separate types for the API return types and then convert, but tbh that seems a bit overkill.

Reply:
Jenkins return  "lastBuild: null".    If I put "lastBuild?: CommonBuild" I cannot test this part as typescript won't let me compile the code.